Modules/System/AL Rest Client/src/ALHttpRequestMessage.Codeunit.al

```diff
@@ -0,0 +1,87 @@
+/// <summary>Holder object for the HTTP request data.</summary>
+codeunit 2352 "AL Http Request Message"
+{
+    var
+        HttpRequestMessage: HttpRequestMessage;
+        Initialized: Boolean;
+        NotInitializedErr: Label 'The Http Request Message has been initialized';
```

Comment:
...has _not_ been...

Reply:
and a dot in the end.

Reply:
Removed the initialization check, so this is not relevant anymore.